argus/backend/plugins/core.py

```diff
@@ -105,7 +105,7 @@ def get_scheduled_assignee(self) -> UUID:
             assignees = ArgusScheduleAssignee.filter(
                 schedule_id=schedule.id
             ).all()
-            assignees_uuids.append(*[assignee.assignee for assignee in assignees])
+            assignees_uuids.extend([assignee.assignee for assignee in assignees])
```

Comment:
What about the other place we have such a code ?

If it's not needed let's clear out as well (in this PR or a follow-up one)

Reply:
Better to clear it out in a follow up pr, the whole directory `argus/db` can be removed.